src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(colors.length);
+        return colors[index];
```

Comment:
Return type String will be perfect

Reply:
maybe I don't understand what you want from me.... 
it will be string or you want 
int index = random.nextInt(colors.length);
String color = colors[index];
return color;